packages/config/config/devices/0x000c/hs-fls100-g2.json

```diff
@@ -0,0 +1,150 @@
+// HomeSeer Technologies HS-FLS100-G2
+// Floodlight Sensor
+{
+	"manufacturer": "HomeSeer Technologies",
+	"manufacturerId": "0x000c",
+	"label": "HS-FLS100-G2",
+	"description": "Floodlight Sensor",
+	"devices": [
+		{
+			"productType": "0x0201",
+			"productId": "0x000c"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "PIR trigger off period",
+			"description": "Period to send Trigger Off command after PIR is triggered.",
+			"unit": "Seconds",
+			"valueSize": 2,
+			"minValue": 8,
+			"maxValue": 720,
+			"defaultValue": 180,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Lux sensor threshold",
+			"description": "Lux level to activate the PIR",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 900,
+			"defaultValue": 50,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Auto Report luminance and temperature interval",
+			"description": "set the interval of periodic lux level report and temperature report to the controller",
+			"unit": "min",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 1440,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "PIR trigger alert",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "disable alert",
+					"value": 0
+				},
+				{
+					"label": "enable alert",
+					"value": 1
+				}
+			]
+		},
+		"5": {
+			"label": "Floodlight control mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "controlled by Z wave controller only",
+					"value": 0
+				},
+				{
+					"label": "controlled by PIR trigger and Lux level, or by Z wave controller",
+					"value": 1
+				}
+			]
+		},
+		"6": {
+			"label": "Lux sensor",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "floodlight is controlled by Lux level and PIR trigger",
+					"value": 0
+				},
+				{
+					"label": "floodlight is controlled by Lux level only, regardless of PIR trigger",
+					"value": 1
+				}
+			]
+		},
+		"7": {
+			"label": "Mesured temperature offset",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 0,
```

Comment:
Which unit? Does this only allow positive offsets?

Reply:
After looking at Z-wave Alliance description closer it looks like it should be -100 to 100 with units of 0.1 °C.


```json
    <ConfigurationParameterExport>
      <Name>Temperature Sensor Offset</Name>
      <Description>0x9C - 0x64 (offset range : -10.0°C ~ +10.0°C)</Description>
      <ParameterNumber>7</ParameterNumber>
      <Size>1</Size>
      <DefaultValue>0</DefaultValue>
      <pType>1</pType>
      <flagReadOnly>false</flagReadOnly>
      <flagReInclude>false</flagReInclude>
      <flagAdvanced>false</flagAdvanced>
      <v3plus>false</v3plus>
      <minValue>0</minValue>
      <maxValue>127</maxValue>
      <ConfigurationParameterValues>
        <ConfigurationParameterValueExport>
          <From>0</From>
          <To>127</To>
          <Description />
        </ConfigurationParameterValueExport>
      </ConfigurationParameterValues>
    </ConfigurationParameterExport>
```